internal/server/server.go

```diff
@@ -105,7 +105,7 @@ func Serve(
 	grpc_health_v1.RegisterHealthServer(grpcServer, healthHandler)
 
 	// init http proxy
-	grpcDialCtx, grpcDialCancel := context.WithTimeout(ctx, time.Second*5)
+	grpcDialCtx, grpcDialCancel := context.WithTimeout(ctx, time.Second*30)
```

Comment:
let's make it configurable with default?

Reply:
done